sdk/core/core-autorest/.vscode/tasks.json

```diff
@@ -0,0 +1,16 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Build Node",
+      "type": "npm",
```

Comment:
Q for myself: how does is work when we are using rush?

Reply:
rushx / rush just run the npm build script, which is all this tells VS code to do. I'm not even sure we need it

Reply:
the only limitation here is it's not going to build deps first